src/USSTree.ts

```diff
@@ -260,7 +260,7 @@ export class USSTree implements IZoweTree<ZoweUSSNode> {
                         baseEncd = values [2];
                     }
                 } catch (error) {
-                    vscode.window.showErrorMessage(error.message);
+                    await errorHandling(error, element.getProfileName(), "Error encountered in flipState.optionalProfiles! ");
```

Comment:
On the same line of thought as my previous comment.
Based on the number of occurrences, I believe we could've localized `Error encountered in:` and handle the function name separately.